contents/docs/libraries/node/index.mdx

```diff
@@ -247,6 +248,51 @@ The same holds for [groups](/manual/group-analytics). If you have a group named
 const flagValue = await client.getFeatureFlag('flag-key', 'user distinct id', {groups:{'organisation': 'google'}, groupProperties:{'organisation': {'is_authorized': True}})
 ```
 
+**Default overridden properties**
+
+Before posthog-node v3.0, by default we added geoip properties to all incoming events, and for feature flag evaluation, based on the IP address of the request. This isn't ideal since it uses your server IP address, rather than the user's, leading to incorrect location resolution.
```

Comment:
```suggestion
Before posthog-node v3.0, we added GeoIP properties to all incoming events by default. We also added them to feature flag evaluation, based on the IP address of the request. This isn't ideal since they are created based on your server IP address, rather than the user's, leading to incorrect location resolution.
```